resources/resource_transformers/babel/babel.go

```diff
@@ -177,6 +200,27 @@ func (t *babelTransformation) Transform(ctx *resources.ResourceTransformationCtx
 		return errors.Wrap(err, errBuf.String())
 	}
 
+	content, err := ioutil.ReadAll(compileOutput)
+	if err != nil {
+		return err
+	}
+
+	if _, err := os.Stat(compileOutput.Name() + ".map"); err == nil {
```

Comment:
Pull the map filename out into a var (you're creating it 3 times).

Reply:
Good idea.

Reply:
@bep NOTE the argument used to be called SourceMaps in this commit, because babel's command line argument is --source-maps and not --source-map. It makes sense, to change it, but this generates no warning or error causing source maps all of a sudden not to work. Not sure what the best thing to do would be, I'll change my references to sourceMaps.